src/typescript/sdk/src/broker-v2/types.ts

```diff
@@ -21,7 +21,8 @@ export type SubscribableBrokerEvents =
   | "MarketLatestState"
   | "GlobalState"
   | "PeriodicState"
-  | "MarketRegistration";
+  | "MarketRegistration"
+  | "Candlestick";
```

Comment:
```suggestion
  | typeof NORMAL_CANDLESTICK_NAME;
```

And just define it in guidgetters (down below) as `NORMAL_CANDLESTICK_NAME = "NormalCandlestick"`